f5/bigip/net/fdb.py

```diff
@@ -64,6 +67,9 @@ def __init__(self, fdb):
         self._meta_data['attribute_registry'] =\
             {u'tm:net:fdb:tunnel:tunnelstate': Tunnel}
         self._meta_data['template_generated'] = True
+        # Setting this here to be explicit, even though it is set via
+        # the super call from its containing object.
+        self._meta_data['icontrol_version'] = '11.5.0'
```

Comment:
If we made the class get its version from its container, and made its container pick up _its_ containers version, then we'd only need to set it explicitly in 'Fdbs'.